Plugins/AWSLambdaPackager/PluginUtils.swift

```diff
@@ -32,7 +32,7 @@ struct Utils {
 
         let fd = dup(1)
         let stdout = fdopen(fd, "rw")
-        defer { fclose(stdout!) }
```

Comment:
we should change ci to use the released swift image here. I think the build issue happens with swift:6.0.0 and swift:6.0.1 but not 6.0-nightly.

Reply:
As much as I understand how it works in Swift-NIO, we can not give the container name as a parameter 
https://github.com/apple/swift-nio/blob/main/.github/workflows/swift_6_language_mode.yml

That would be a change request for the Swift-NIO project

Reply:
The problem with `!` is not in the CI, it crashes the plugin when using it.  We can not force unwrap, there are cases where stdout is nil